gallery/src/components/NxStatefulIconDropdown/NxStatefulIconDropdownPage.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxTextLink, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+
+import NxStatefulIconDropdownExample from './NxStatefulIconDropdownExample';
+
+const nxStatefulIconDropdownExampleCode = require('./NxStatefulIconDropdownExample?raw');
+
+const NxStatefulDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A variation of the icon only dropdown menu component which uses a single icon as its toggle.{' '}
+        <NxCode>NxStatefulIconDropdown</NxCode> shares all of the same props as{' '}
+        <NxTextLink href="#/pages/NxIconDropdown">NxIconDropdown</NxTextLink>{' '}
+        as well as the ones listed below.
```

Comment:
> > I'll admit to some confusion though since the NxStatefulDropdown docs show a great deal of repetition between it and the non-stateful version.
> 
> Yeah it ideally probably shouldn't repeat the parts that are the same.

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/5ccd3d4f4a93b6b9f09f76f799c1a9e0e56c45b9